File: OOAD/employee-max-salary-app/src/loader/WebLoader.java
package loader;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.net.URL;

public class WebLoader implements ILoadable{
	URL urlReader;
	BufferedReader bufferedReader;
	
	@Override
	public String loadFile() throws IOException {
		urlReader=new URL("https://swabhav-tech.firebaseapp.com/emp.txt");
		bufferedReader = new BufferedReader(new InputStreamReader(urlReader.openStream()));
		String currentLine = "";
		String finalString = "";

		while ((currentLine = bufferedReader.readLine()) != null) {
			finalString = finalString + currentLine + "/";
		}
		//System.out.println(finalString);
		return finalString;
	}

}

File: C#/OOP/Employee_max_Salary-App/Employee_max_Salary-App/WebLoader.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Net;

namespace Employee_max_Salary_App
{
    class WebLoader : ILoadable
    {
        string text;
        public string LoadFile()
        {
            WebClient wc = new WebClient();
            try
            {
                string result = wc.DownloadString("https://swabhav-tech.firebaseapp.com/emp.txt");
                Console.WriteLine(result);
            }
            catch (WebException we) { Console.WriteLine(we.ToString()); }
            
            return text;
        }
    }
}
